main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+)
```

Comment:
Sugestão sobre a linha 7, que há espaço desnecessário
```suggestion
import (
	"fmt"
	"net/http"
	"strconv"
	"github.com/gofiber/fiber/v2"
)
```